node_modules/gh-users/tests/test-users.js

```diff
@@ -149,5 +149,18 @@ describe('Users', function() {
                 });
             });
         });
+
+        /**
+         * Test that verifies that a user cannot use another user's email address
+         */
+        it('verify a user cannot use another user\'s email address', function(callback) {
+            TestsUtil.generateTestUsers(global.tests.apps.cam2014, 2, false, function(simon, nico) {
+
+                // Nico shouldn't be able to re-use Simon's email address
```

Comment:
Creating users is one of those areas that hasn't received much test love yet somehow ;-)